about-me.md

```diff
@@ -1,9 +1,9 @@
 # About Me
 
-![Picture of me]()
+![Picture of me](https://avatars1.githubusercontent.com/u/40181132?s=460&v=4)
 
-Hi, I'm ____________. I'm a ________ at *__________*.
+Hi, I'm Charlene B. Trivedi. I'm a content strategist at *OPM*.
```

Comment:
Charlene? Is that an error?

Reply:
Ah, yeah. Charlene is my legal name. Updated for clarity, thanks for catching that!